ynr/apps/resultsbot/election_id_to_url.csv

```diff
@@ -103,3 +103,33 @@ local.eastleigh.2019-05-02,https://meetings.eastleigh.gov.uk/mgWebService.asmx/G
 local.west-sussex.2019-05-02,https://westsussex.moderngov.co.uk/mgWebService.asmx/GetElectionResults?lElectionId=1
 local.north-west-leicestershire.2019-05-02,https://minutes-1.nwleics.gov.uk/mgWebService.asmx/GetElectionResults?lElectionId=4
 local.stoke-on-trent.2019-05-02,http://www.moderngov.stoke.gov.uk/mgWebService.asmx/GetElectionResults?lElectionId=17
```

Comment:
Is it safe to delete these?

Reply:
So I've used these in the past to generate new URLs, or at least have a good idea of where to start. This file isn't the best place for it, but let's keep it here for the moment until we have a proper place for tracking this info